data/borderlines.json

```diff
@@ -197,6 +212,21 @@
           "name": "Everest Fun Park & Resort Pvt. Ltd.",
           "latitude": 26.6431722,
           "longitude": 87.6181256
+        },
+        {
+          "name": "Urlabari Multiple Campus",
+          "longitude": 26.6651,
+          "latitude": 87.62
+        },
+        {
+          "name": "Shree Saraswati Secondary School",
```

Comment:
This isn't in urlabari still.

Reply:
Is it? On Google Maps, it shows this address.

> MHMP+7W, Urlabari 56600, Nepal

Reply:
Do you still want me to make a change? I'll add another landmark then

Reply:
> This isn't in urlabari still.

Changed it to Madan Bhandari Hospital & Trauma Center.